app/jobs/schedule_server_load_readings_job.rb

```diff
@@ -0,0 +1,22 @@
+class ScheduleServerLoadReadingsJob < ApplicationJob
```

Comment:
Would `ServerLoadReadingJob` be a better name for this? Since we are not scheduling any jobs here, but executing the jobs directly here.